dockerfiles/Dockerfile

```diff
@@ -32,7 +32,10 @@ RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
 
 RUN pip3 install --no-cache-dir --upgrade pip
 
-COPY requirements /tmp/requirements
-RUN pip3 install --no-cache-dir -r /tmp/requirements/docker.txt
+WORKDIR /tmp
+
+RUN curl -O https://raw.githubusercontent.com/readthedocs/readthedocs.org/master/requirements/pip.txt
+RUN curl -O https://raw.githubusercontent.com/readthedocs/readthedocs.org/master/requirements/docker.txt
```

Comment:
These two lines are going to be a nightmare for testing different versions of things.

Reply:
I'm not following your point here. What would be the problem?

Reply:
I think David means we should install deps the first time the container is run (or when init=true) instead of installing them when building the image.